.github/workflows/test.yml

```diff
@@ -1,6 +1,13 @@
 name: Test
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
I didn't even know that we're using the `dev` branch. I'd prefer merging everything in `main`, but if you prefer a different model, we should describe it somewhere.